packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/bitmap.dart

```diff
@@ -306,10 +329,100 @@ class BitmapDescriptor {
     );
   }
 
-  final Object _json;
-
   /// Convert the object to a Json format.
-  Object toJson() => _json;
+  Object toJson();
+}
+
+/// A BitmapDescriptor using the default marker.
+class DefaultMarker extends BitmapDescriptor {
+  /// Provide an optional [hue] for the default marker.
+  const DefaultMarker([this.hue]) : super._();
```

Comment:
Please use named parameters so that we can change the parameter set in the future without breaking changes (or being stuck with a growing positional list, which doesn't scale).